recipes/pyftpdlib/meta.yaml

```diff
@@ -0,0 +1,55 @@
+{% set name = "pyftpdlib" %}
+{% set version = "1.5.0" %}
+{% set sha256 = "88c1dc3daa8de9ff5b25fd57ae8cc4390b5d034f87f2396ebd93eeb8f6879972" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: release-{{ version }}.tar.gz
+  url: https://github.com/giampaolo/pyftpdlib/archive/release-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+  entry_points:
+    - pyftpdlib = clcache:main
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - python
```

Comment:
Another fact I noticed about `pysendfile` is that it is not supported on windows. Its [readme](https://github.com/giampaolo/pysendfile) says:

> This module works with Python versions from 2.5 to 3.4. The supported platforms are:
> - Linux
> - Mac OSX
> - FreeBSD
> - Dragon Fly BSD
> - Sun OS
> - AIX (not properly tested)

So no windows there, nor Python 3.5.